src/transaction-types.ts

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable no-restricted-globals */
```

Comment:
This can be removed now.

Reply:
[67896a9](https://github.com/MetaMask/utils/pull/74/commits/67896a91770067d8e09151cc3ff1e124fe1ddc84)